frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss

```diff
@@ -0,0 +1,3 @@
+.ceph-capacity-card__dashboard-card {
```

Comment:
Should we consider a general solution that can be used in other contexts? I have a feeling we'll see this in more places as we build out these dashboards.